Automattic/wp-calypso
# Jetpack Connect: Activate Jetpack Manage

Activating Jetpack's Manage module after the authorization is complete, and a user clicks to browse upgrades.

To test:
- Apply this PR to a Jetpack site: https://github.com/Automattic/jetpack/pull/3733
- Turn off 'Manage' and disconnect Jetpack
- Apply this PR to your local Calypso
- In your console, run `localStorage.setItem( 'debug', 'calypso:jetpack-connect:*' )`
- Visit calypso.localhost:3000/jetpack/connect, and go through the steps to connect your Jetpack site
- In the final step click 'Browse Available Upgrades'
- Note that the debug messages in your console say that Manage is activated
- Check your site and make sure Manage is activated

Note: The UI is still primitive. @rickybanister has some mocks that will make it clear to the user that they are turning on the Manage module

CC @johnHackworth
